lib/helpers/config.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const config = {
+  nodeEnv: process.env.NODE_ENV,
```

Comment:
I also thought of this and it will also help to have the same var name used in the code as it is defined in the `.env`. I just kept it this way because it's the way Dan initially wrote it (former dev in the team) but, yes, I think I'll update it.